hs_core/models.py

```diff
@@ -2252,16 +2277,71 @@ def has_required_content_files(self):
         else:
             return True
 
+    @property
+    def readme_file(self):
+        """Returns a resource file that is at the root with a file name of either
+        'readme.txt' or 'readme.md' (filename is case insensitive). If no such file then None
+        is returned. If both files exist then resource file for readme.md is returned"""
+
+        res_files_at_root = [f for f in self.files.all() if f.file_folder is None]
```

Comment:
We don't want to look through all the files for a readme each time the page loads.  These types of things add up and contribute to the long page loading times.